src/mitsubishi2mqtt/mitsubishi2mqtt.ino

```diff
@@ -1292,6 +1292,9 @@ void hpStatusChanged(heatpumpStatus currentStatus) {
     // send room temp, operating info and all information
     heatpumpSettings currentSettings = hp.getSettings();
 
+	const size_t bufferSizeInfo = JSON_OBJECT_SIZE(10);
+	StaticJsonDocument<bufferSizeInfo> rootInfo;
```

Comment:
This needed to be added due to an issue with Pull Request #105. That update caused the "mode" and "action" JSON elements to show null values.